cmd/ftl/cmd_schema_diff.go

```diff
@@ -57,6 +83,31 @@ func (d *schemaDiffCmd) Run(ctx context.Context, currentURL *url.URL) error {
 	return nil
 }
 
+func localSchema(ctx context.Context, projectConfig projectconfig.Config) (*schema.Schema, error) {
+	pb := &schema.Schema{}
+	found := false
+	tried := ""
+	modules, err := buildengine.DiscoverModules(ctx, projectConfig.AbsModuleDirs())
+	if err != nil {
+		return nil, fmt.Errorf("failed to discover modules %w", err)
+	}
+	for _, moduleSettings := range modules {
+		path := filepath.Join(moduleSettings.Config.Abs().Dir, ".ftl", "schema.pb")
```

Comment:
1. Store the result of `.Abs()` in a variable outside the loop.
2. Use `ModuleConfig.Schema` rather than joining manually.

Reply:
(`.ftl/schema.pb` varies per language runtime)

Reply:
The .Abs is per module, how can I move it out of the loop? The call to .Abs outside the loop does not seem to result in modules that are using the Abs path.